xs/src/slic3r/GUI/3DScene.hpp

```diff
@@ -41,6 +41,9 @@ class GLVertexArray {
         this->norms.push_back(z);
     };
     void load_mesh(const TriangleMesh &mesh);
+    Pointf3 getPoint(int i){
```

Comment:
Please match the style of the other functions in this class.

I'd also prefer using ->at() instead so that it'll throw an exception if somehow it goes out of bounds.